Tasks/Maven/task.json

```diff
@@ -16,7 +16,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 69
+        "Patch": 70
```

Comment:
This is not an accurate version - current patch is 71. I'll make sure to change it before merging.